Documentation/install/upgrade.rst

```diff
@@ -291,7 +291,34 @@ Annotations:
    upgrade. Connections should successfully re-establish without requiring
    clients to reconnect.
 
-.. _1.7_upgrade_notes:
```

Comment:
It looks like this is removing the `_1.7_update_notes` reference. It should probably be retained.